js/views/modals/Settings/Advanced.js

```diff
@@ -0,0 +1,103 @@
+import app from '../../../app';
+import loadTemplate from '../../../utils/loadTemplate';
+import baseVw from '../../baseVw';
+import $ from 'jquery';
+
+const siblingsToggle = '[data-next-siblings-toggle]';
+const siblingsToggleInputs = `${siblingsToggle} input[type="radio"]`;
+
+export default class extends baseVw {
+  constructor(options = {}) {
+    super({
+      className: 'settingsAdvanced',
+      ...options,
+    });
+
+    this.settings = app.settings.clone();
+
+    this.listenTo(this.settings, 'sync', () => app.settings.set(this.settings.toJSON()));
+  }
+
+  get events() {
+    return {
+      [`change ${siblingsToggleInputs}`]: 'handleNextSiblingsToggle',
+    };
+  }
+
+  handleNextSiblingsToggle(event) {
+    const t = event.target;
+    this.applyNextSiblingsToggle(t, t.value);
+  }
+
+  applyNextSiblingsToggle(t, val) {
+    const p = $(t).parents(siblingsToggle);
+    if (val === 'true') {
+      p.addClass('showNextSiblings');
+    } else {
+      p.removeClass('showNextSiblings');
+    }
+  }
+
+  getFormData(subset = this.$formFields) {
+    return super.getFormData(subset);
+  }
+
+  saveServer() {
+    const formData = this.getFormData();
+    this.settings.set(formData);
+    return this.settings.save();
+  }
+
+  saveLocal() {
+    const localData = this.getFormData(this.$localFields);
+    app.localSettings.set(localData);
+    return app.localSettings.save();
+  }
+
+  reportSave(save) {
+    this.trigger('saving');
+
+    if (!save) {
+      // client side validation failed
+      this.trigger('saveComplete', true);
+    } else {
+      this.trigger('savingToServer');
+
+      save
+        .done(() => {
+          this.trigger('saveComplete');
+        })
+        .fail((...args) => {
+          this.trigger('saveComplete', false, true,
+            args[0] && args[0].responseJSON && args[0].responseJSON.reason || '');
+        });
+    }
+
+    this.render();
+    const $firstErr = this.$('.errorList:first');
+    if ($firstErr.length) $firstErr[0].scrollIntoViewIfNeeded();
+  }
+
+  save() {
+    this.reportSave(this.saveLocal() && this.saveServer());
+  }
+
+  render() {
+    loadTemplate('modals/settings/advanced.html', (t) => {
+      this.$el.html(t({
+        errors: this.settings.validationError || {},
+        ...this.settings.toJSON(),
+        ...app.localSettings.toJSON(),
+      }));
+
+      this.$formFields = this.$('select[name], input[name], textarea[name]').
+        not('[data-persistence-location="local"]');
+      this.$localFields = this.$('[data-persistence-location="local"]');
+      this.$el.find(`${siblingsToggleInputs}:checked`).
```

Comment:
You don't have to make this change, but as an FYI, backbone view's give you a shortcut for `this.$el.find`.

`this.$el.find('<selector>')` ==> `this.$('<selector>')`